modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -676,50 +674,92 @@ PERF_TEST_P_(CmpWithScalarPerfTest, TestPerformance)
 
 PERF_TEST_P_(BitwisePerfTest, TestPerformance)
 {
-    bitwiseOp opType = get<0>(GetParam());
-    cv::Size sz = get<1>(GetParam());
-    MatType type = get<2>(GetParam());
-    cv::GCompileArgs compile_args = get<3>(GetParam());
+    MatType   type           = -1;
+    bitwiseOp opType         = AND;
+    bool      testWithScalar = false;
+    cv::Size sz;
+    cv::GCompileArgs compile_args;
+
+    std::tie(opType, testWithScalar, sz, type, compile_args) = GetParam();
 
     initMatsRandU(type, sz, type, false);
 
     // G-API code & corresponding OpenCV code ////////////////////////////////
     cv::GMat in1, in2, out;
-    switch (opType)
+    if( testWithScalar )
     {
-    case AND:
-    {
-        out = cv::gapi::bitwise_and(in1, in2);
-        cv::bitwise_and(in_mat1, in_mat2, out_mat_ocv);
-        break;
+        cv::GScalar sc1;
+        switch (opType)
+        {
+        case AND:
+        {
```

Comment:
you don't need extra `{}` under every case as you don't declare any variables there.

Reply:
Thanks! fixed